pkgs/initiator/initiator.go

```diff
@@ -96,8 +97,11 @@ func GenerateAggregatesKeyshares(keySharesArr []*wire.KeySharesCLI) (*wire.KeySh
 }
 
 // New creates a main initiator structure
-func New(operators wire.OperatorsCLI, logger *zap.Logger, ver string) (*Initiator, error) {
+func New(operators wire.OperatorsCLI, logger *zap.Logger, ver string, certs []string) (*Initiator, error) {
 	client := req.C()
+	// set CA certificates if any
+	client.SetRootCertsFromFile(certs...)
```

Comment:
even though it works with an empty slice, we should probably only call it `if len(certs) > 0`
otherwise a library upgrade might break it